package.json

```diff
@@ -114,6 +114,7 @@
     "publish:ci": "node utils/pre-publish-check-ci.js && git diff --stat --exit-code HEAD && yarn build && yarn extract-strings && changeset publish",
     "sloc": "sloc packages --exclude node_modules",
     "test": "yarn jest",
+    "test:raw": "cross-env GLOBAL_CONSOLE_MOCK=false yarn jest",
```

Comment:
Maybe `test:no-console-mock`? Makes the intent clearer.

Reply:
I had that originally, but it feel like a lot to type.  I'll switch it.